_dashboards/discover/multi-data-sources.md

```diff
@@ -0,0 +1,134 @@
+---
+layout: default
+title: Multiple data sources
+parent: Discover
+nav_order: 5
+---
+
+Multiple data sources is an experimental feature released in OpenSearch 2.4. We do not recommend using it in a production environment. For updates on the feature’s progress or to leave feedback on improving the feature, see the [OpenSearch Forum discussion](https://forum.opensearch.org/t/feedback-experimental-feature-connect-to-external-data-sources/11144).
+{: .note }
+
+# Multiple data sources
+
+You can add multiple data sources to a single dashboard. OpenSearch Dashboards allows you to dynamically manage data sources, create index patterns based on those data sources, and execute queries against a specific data source and then combine visualizations in one dashboard.
+
+In this tutorial we provide the steps for enabling the data sources setting in Dashboards; adding credentials, data source connections, and index patterns; and combining visualizations in a single dashboard.
+
+## Try out Data Sources in your local environment
+
+This tutorial uses a preconfigured data source and index pattern, and you aren’t required to configure settings. However, you’ll need to enable the multiple data sources setting in the configuration file before before getting started with exploring this feature.
+{: .note }
+
+Multiple data sources is an experimental feature that can't be deployed into production, but you can try it out with a sample data source and a sample index pattern. Before getting started, you must first edit the YAML configuration. The following section provides the steps to enable the feature.
```

Comment:
Should we maybe use language to have people try it and to leave feedback? Maybe like "we encourage you to try out / test this feature and provide feedback" etc?